src/pages/RealEstateDetail/Chart/lineChart.js

```diff
@@ -0,0 +1,15 @@
+export const makeLineChartData = () => {
+  const line = {
+    labels: ['#1', '#2', '#3', '#4', '#5', '#6', '#7', '#8'],
+    datasets: [
+      {
+        label: '신용자 등급 정보',
+        data: [900, 920, 880, 890, 850, 870, 880, 890, 920, 920, 780, 800],
+        fill: false,
+        borderColor: 'rgb(75, 192, 192)',
+        tension: 0.1,
+      },
+    ],
+  };
+  return line;
+};
```

Comment:
```suggestion
export const makeLineChartData = () => {
    return {
    labels: ['#1', '#2', '#3', '#4', '#5', '#6', '#7', '#8'],
    datasets: [
      {
        label: '신용자 등급 정보',
        data: [900, 920, 880, 890, 850, 870, 880, 890, 920, 920, 780, 800],
        fill: false,
        borderColor: 'rgb(75, 192, 192)',
        tension: 0.1,
      },
    ],
  };
};
```
변수 선언해서 할당하는 과정은 필요없어보이네요! 바로 리턴해도 되보입니다!

Reply:
바로 리턴으로 변경했습니다.